PWGDQ/Core/MCSignalLibrary.h

```diff
@@ -121,6 +121,11 @@ MCSignal* o2::aod::dqmcsignals::GetMCSignal(const char* name)
     signal = new MCSignal(name, "Pi0", {prong}, {-1});
     return signal;
   }
+  if (!nameStr.compare("SM")) {
```

Comment:
Please use a more self-explanatory name. SM, from "same mother" does not give any hint and its confusing. Many multiple prong signals can have the requirement of having a common mother, while this particular one is just a one generation and single prong signal, where the single mother requirement is meaningless. 
For this, see also my comment to the MCProng defintion of code 600.